test/components/ProjectListSpec.js

```diff
@@ -1,11 +1,8 @@
 import test from 'tape'
 import React from 'react'
 import { MemoryRouter } from 'react-router-dom'
-import { createStore } from 'redux'
 
-import {
-  mount
-} from 'enzyme'
+import { mount } from 'enzyme'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'enzyme'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->